### PR TITLE
Tidying up the dependencies and the indentation consistency of the project

### PR DESCRIPTION
**Pull request by BastiDood**
Hello, there. It's me again _back at it again with tidying up_. This pull request includes a bunch of formatting fixes for the project. I saw that you were using two spaces as indentation in some files while it was four for others. I decided to bring the project together under one roof by using two spaces as the official indentation for all files.

I also cleaned up the dependencies a bit. I first noticed this strange module in the `dependencies` named [`os`](https://www.npmjs.com/package/os), which was basically a wrapper over Node.js' native implementation of the `os` module. I mean the entire module literally only has a single line of code in its [`index.js`](https://github.com/DiegoRBaquero/node-os/blob/master/index.js#L1), which consists of redundantly exporting the `os` module of Node.js. I thought that maybe it would be worth removing this unnecessary module in the dependencies.

I also updated the dependencies to the latest minor versions. Apparently, Handlebars had a "Prototype Pollution" vulnerability in the previous version, so I took the opportunity to update all of the modules. It was simply me running `npm update`.

I actually considered upgrading to Electron v5, but I figured that it might be too soon for that considering that the latest version is at 5.0.2. There may be some unwanted bugs and consequences in the latest major version.

Anyway, that concludes this pull request. If you don't mind, I have a lot of plans for a major refactoring of the code base. I plan on implementing a more "object-oriented approach" to the architecture of this small app by encapsulating functionalities and features into classes for the sake of maintainability.

I also have plans in refactoring the CSS in such a way that takes advantage of CSS variables so that it is easier to apply themes. We don't have to worry about browser support anyway since this is an Electron app.

In other words, I have plans to make this app more maintainable for the future so that when the dev.to API officially becomes available, it will be easy to refactor the code from a crawler-based architecture to an external API-compliant architecture... if that makes sense? I'm not really sure about the proper terminology, but I'm sure you get the point.

### CODE REVIEW

**Comment by sarthology** [comment]
Wow, loved to see that you back. I am up for anything you purpose man. Let's do it. I think it's time to add you as a contributor. 😊
